ndsl/constants.py

```diff
@@ -76,70 +78,71 @@ class ConstantVersions(Enum):
 # Physical constants
 #####################
 if CONST_VERSION == ConstantVersions.GEOS:
-    RADIUS = 6.371e6
-    PI = 3.14159265358979323846
-    OMEGA = 2.0 * PI / 86164.0
-    GRAV = 9.80665
-    RGRAV = 1.0 / GRAV
-    RDGAS = 8314.47 / 28.965
-    RVGAS = 8314.47 / 18.015
-    HLV = 2.4665e6
-    HLF = 3.3370e5
-    KAPPA = RDGAS / (3.5 * RDGAS)
+    RADIUS = Float(6.371e6)
+    PI_8 = np.float64(3.14159265358979323846)
+    PI = Float(PI_8)
+    OMEGA = Float(2.0) * PI / Float(86164.0)
+    GRAV = Float(9.80665)
+    RGRAV = Float(1.0) / GRAV
+    RDGAS = Float(8314.47) / Float(28.965)
+    RVGAS = Float(8314.47) / Float(18.015)
+    HLV = Float(2.4665e6)
+    HLF = Float(3.3370e5)
+    KAPPA = RDGAS / (Float(3.5) * RDGAS)
     CP_AIR = RDGAS / KAPPA
-    TFREEZE = 273.15
-    SAT_ADJUST_THRESHOLD = 1.0e-6
+    TFREEZE = Float(273.16)
+    SAT_ADJUST_THRESHOLD = Float(1.0e-6)
 elif CONST_VERSION == ConstantVersions.GFS:
```

Comment:
Let's change this `.GFS` nomenclature to `.UFS` to make it clear that it's for that modeling system.